### PR TITLE
Update secret

### PR DESCRIPTION
**Pull request by oyvindeide**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
+    env: TOXENV=formatting
+  - python: 3.7
+    env: TOXENV=docs
 before_install:
-    - wget https://chromedriver.storage.googleapis.com/$(wget https://chromedriver.storage.googleapis.com/LATEST_RELEASE -q -O -)/chromedriver_linux64.zip
-    - unzip chromedriver_linux64.zip
-    - export PATH=$PATH:$PWD
-
+- wget https://chromedriver.storage.googleapis.com/$(wget https://chromedriver.storage.googleapis.com/LATEST_RELEASE
+  -q -O -)/chromedriver_linux64.zip
```
Comment (lars-petter-hauge) [inline .travis.yml:20]: isn't this supposed to be on the same line?

Comment (oyvindeide) [reply .travis.yml:20]: Travis auto-formatted it for me, but can change it back if you want

File: .travis.yml [hunk 2]
```diff
@@ -1,42 +1,37 @@
...
-      env: TOXENV=py38
-    - python: 3.7
-      env: TOXENV=linting
-    - python: 3.7
-      env: TOXENV=formatting
-    - python: 3.7
-      env: TOXENV=docs
-
-
+  - python: 3.6
```
Comment (lars-petter-hauge) [inline .travis.yml:6]: I don't see why you would need to change the indentation here? if any, I think it made it worse :P. From what I could find online, others agree.

Comment (oyvindeide) [reply .travis.yml:6]: Same reason as above, I ran it through the travis cli, and it auto-formatted it,I have no strong preference

File: .travis.yml (hunk 1 above)
Comment (lars-petter-hauge) [reply .travis.yml:20]: Travis auto-formatted it?

File: .travis.yml [hunk 3]
```diff
@@ -1,42 +1,37 @@
...
 install:
-    - pip install tox
-
+- pip install tox
 script:
-    - tox
-
+- tox
 deploy:
+  matrix:
```
Comment (lars-petter-hauge) [inline .travis.yml:28]: suppose then this is something that is redundant, but travis would like to have it in?

File: .travis.yml (hunk 1 above)
Comment (lars-petter-hauge) [reply .travis.yml:20]: I think I would prefer to change it back. Although most were indents, I also saw an added `matrix`.

File: .travis.yml (hunk 3 above)
Comment (oyvindeide) [reply .travis.yml]: Correct!

**Pull request merged**